prowler/providers/kubernetes/services/core/core_service.py

```diff
@@ -0,0 +1,83 @@
+from typing import List, Optional
+
+from kubernetes import client
+from pydantic import BaseModel
+
+from prowler.lib.logger import logger
+from prowler.providers.kubernetes.lib.service.service import KubernetesService
+
+
+################## Core ##################
+class Core(KubernetesService):
+    def __init__(self, audit_info):
+        super().__init__(audit_info)
+        self.client = client.CoreV1Api(self.api_client)
+
+        self.pods = []
+        self.__get_pods__()
+
+    def __get_pods__(self):
+        try:
+            pods = self.client.list_pod_for_all_namespaces()
+            for pod in pods.items:
+                pod_containers = []
```

Comment:
The same here, better to use a map with UIDs https://kubernetes.io/docs/concepts/overview/working-with-objects/names/

Reply:
Done